src/tools/schema2kotlin.ts

```diff
@@ -107,7 +126,7 @@ import arcs.core.storage.api.toPrimitiveValue`}
     }
     return `
 ${this.getHandlesClassDecl(particleName, specDecls)} {
-    ${handleDecls.join('\n    ')} 
+    ${handleDecls.join('\n    ')}
```

Comment:
If you can bear with an advertisement: 
You might enjoy using `ktUtils.joinWithIndents` here.

Reply:
Thanks Alex, I will elsewhere, but these need to be on different lines (and not be comma separated.